projects/plugins/jetpack/to-test.md

```diff
@@ -1,91 +1,36 @@
-## Jetpack 12.1
+## Jetpack 12.2
 
 ### Before you start:
 
 - **At any point during your testing, remember to [check your browser's JavaScript console](https://wordpress.org/support/article/using-your-browser-to-diagnose-javascript-errors/#step-3-diagnosis) and see if there are any errors reported by Jetpack there.**
 - Use the "Debug Bar" or "Query Monitor" WordPress plugins to help make PHP notices and warnings more noticeable and report anything of note you see.
 
-### Visual refresh
-Jetpack 12.1 brings a visual refresh to multiple pages of the admin area. Please visit them in your testing and make sure nothing looks off.
+### AI Assistant:
+Jetpack 12.2 brings a new AI Assistant block. Given Automattic's current focus on AI, this should be tested. See p1HpG7-mHr-p2 for details.
```

Comment:
Can we please add these instructions inline here?

Reply:
At the moment they aren't even finished writing them. :shrug: